src/main/Main.java

```diff
@@ -1,7 +1,32 @@
+import java.sql.Connection;
+
+import constants.ResponseStatus;
+import dto.LoginDTO;
+import dto.ResponseDTO;
+import dto.SignUpDTO;
+import dto.UserProfileDTO;
+import views.AuthView;
+import views.UserProfileView;
+
 public class Main {
 	public static void main(String[] args) {
 		try {
-			Database.connection();
+			LoginDTO log_in = new LoginDTO();
+			SignUpDTO sign_up = new SignUpDTO();
+			UserProfileDTO user_profile = new UserProfileDTO();
+			Connection con = Database.connection();
+			AuthView view = new AuthView(con);
+			UserProfileView user_profile_view = new UserProfileView(con);
+			ResponseDTO<Object> response = view.Auth_view(sign_up, log_in);
+			if(response.getMessage() == "Sign up successfully!") {
```

Comment:
Why dont check by response status

Reply:
Because if I use response status. When u log in successfully, it will let u insert your user profile

Reply:
Im still not understand, explain again and make it more clearly, pls